patches/minecraft/net/minecraft/block/BlockCactus.java.patch

```diff
@@ -9,7 +9,26 @@
  {
      public static final PropertyInteger field_176587_a = PropertyInteger.func_177719_a("age", 0, 15);
      protected static final AxisAlignedBB field_185593_b = new AxisAlignedBB(0.0625D, 0.0D, 0.0625D, 0.9375D, 0.9375D, 0.9375D);
-@@ -110,8 +110,8 @@
+@@ -48,7 +48,8 @@
+             if (i < 3)
+             {
+                 int j = ((Integer)p_180650_3_.func_177229_b(field_176587_a)).intValue();
+-
```

Comment:
@LexManos I am doing my best to follow the "how to contribute" guidelines.  In this case here I put the new line of code onto an existing blank line.  If that causes a patch change that is undesirable, then the how-to wiki page needs to be updated to reflect that case.  It currently has the insert of a new statement between two lines with _no_ blank line between them stating "don't add blank lines."  I used one that was there, if that was wrong, then I have been given bad instructions.